### ISSUE TITLE
Consider adding actors field to families

### ISSUE THREAD

**Issue by yankovs** [danielplohmann/mcrit#57]
Hey :)!

It would be cool if the family summary would include the actors associated with it. Somewhat similar to the way Malpedia has this info present:
<img width="492" alt="image" src="https://github.com/danielplohmann/mcrit/assets/49753115/062b7297-9c55-4137-8ada-86712c3fb0d2">

It does change the DB's overall schema I guess, but since it's only addition of data on top of existing JSONs I think it's ok. What do you think?

**Comment by danielplohmann** [comment]
Yeah, this will become immediately possible once tagging is added, see #53.
